crates/bitwarden/Cargo.toml

```diff
@@ -13,12 +13,16 @@ edition = "2021"
 rust-version = "1.57"
 
 [features]
-default = ["secrets"]
+default = ["secrets", "use_native_tls"]
 
 secrets = []                    # Secrets manager API
 internal = []                   # Internal testing methods
 mobile = ["uniffi", "internal"] # Mobile-specific features
 
+# TLS backend selection
+use_native_tls = ["reqwest/native-tls"]
+use_rustls = ["reqwest/rustls-tls"]
```

Comment:
We should look into using https://github.com/rustls/rustls-platform-verifier for rustls

Reply:
Implemented support for it everywhere but android, where I chose rustls-webpki instead. We can change it later once the Android setup is easier.